building/build-debs/homeworld-admin-tools/src/authority.py

```diff
@@ -43,8 +51,8 @@ def generate() -> None:
                 # public keys; copy over without encryption
                 util.copy(os.path.join(certdir, filename), os.path.join(cryptdir, filename))
             else:
-                # private keys; encrypt when copying
-                keycrypt.gpg_encrypt_file(os.path.join(certdir, filename), os.path.join(cryptdir, filename))
+                # private keys; encrypt when copying, and rename encrypted version for clarity.
+                keycrypt.gpg_encrypt_file(os.path.join(certdir, filename), os.path.join(cryptdir, encrypted_name(filename)))
```

Comment:
Didn't you rename encrypted_name? Unless I'm missing something, there's no way this will work.